system/jlm/playlist.xml

```diff
@@ -42,6 +42,12 @@
 	<!-- Temporarily excluding from z/OS due to : jdk11-zos/issues/567 -->
 	<test>
 		<testCaseName>TestJlmRemoteClassAuth</testCaseName>
+		<disable>
+			<comment>Disabled on aix for https://github.com/eclipse-openj9/openj9/issues/11557</comment>
+			<platform>ppc64_aix</platform>
+			<impl>openj9</impl>
+			<impl>ibm</impl>
```

Comment:
This format is not correct. 
- There is no `<disables>`. Please see https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#manually-exclude-a-test-target
- Cannot use two impl in one `<disable>`

Reply:
We do not need `<impl>ibm</impl>` as there is no aix build uses impl=ibm.

Reply:
Thanks! Fixed! Please review!